Dockerfile

```diff
@@ -55,4 +55,4 @@ USER rails:rails
 # Start the server by default, this can be overwritten at runtime
 EXPOSE 3000
 # Start the server
-CMD ["bin/rails", "server", "-b", "0.0.0.0"]
\ No newline at end of file
+CMD ["bin/http.sh"]
```

Comment:
Is this change just to make debugging easier?

Reply:
oh, i'm guessing it is to be able to copy over it with ssh.sh?

Reply:
yes, the ssl config involves some env vars, etc, so this made it simpler -- just did the same with http